README.md

```diff
@@ -134,6 +134,9 @@ Determines whether the `cmi.completion_status` is set to "completed" if the asse
 #### \_showCookieLmsResetButton (boolean):
 Determines whether a reset button will be available to relaunch the course and optionally clear tracking data (scorm_test_harness.html only). The default is `false`.
 
+#### \_shouldPersistCookieLMSData (boolean):
+Determines whether to persist the cookie data over browser sessions (scorm_test_harness.html only). The default is `false`.
```

Comment:
Can we default this to `true` to preserve existing behaviour? Also it was [mentioned ](https://github.com/adaptlearning/adapt-contrib-spoor/pull/248#discussion_r1016496365) scorm_test_harness is sometimes used in production so it would be expected that data is persisted.